src/vs/workbench/contrib/extensions/browser/extensionEditor.ts

```diff
@@ -945,22 +945,25 @@ export class ExtensionEditor extends EditorPane {
 		append(moreInfoContainer, $('.additional-details-title', undefined, localize('Marketplace Info', "More Info")));
 		const moreInfo = append(moreInfoContainer, $('.more-info'));
 		if (gallery) {
+			const pub = new Date(gallery.releaseDate);
+			const last_rel = new Date(gallery.lastUpdated);
 			append(moreInfo,
 				$('.more-info-entry', undefined,
 					$('div', undefined, localize('published', "Published")),
-					$('div', undefined, new Date(gallery.releaseDate).toLocaleString(language, { hourCycle: 'h23' }))
+					$('div', undefined, pub.toISOString().slice(0, 10).concat(', ' + pub.toLocaleTimeString(language, { hourCycle: 'h23' })))
```

Comment:
`toISOString()` is returning the UTC Time. I would recommend to construct the string by getting each value (day, month, year...)

Reply:
Got you! Would implement that and commit when done

Reply:
> `toISOString()` is returning the UTC Time. I would recommend to construct the string by getting each value (day, month, year...)

Hey! I tried working something. Do let me know if it works!

Reply:
@sandy081 Worked on another one as well. Do let me know whichever solution suits!